### ISSUE TITLE
Intro: citations

### ISSUE THREAD

**Issue by donovanr** [AllenCellModeling/pytorch_integrated_cell#58]
## Issue summary
Add some citations and buffer text to the intro

## Details
> Convolutional neural networks are becoming more commonly used for image analysis in cell biology. Most applications use CNN to perform classification tasks. These generally fall into two categories: 
> 1) Pixel-by-pixel classification such as for segmentation (D. A. van Valen et al., 2016, PLoS Comp Biol 12: e1005177), for label-free prediction (C. Ounkomol et al., Ref. 6; E. M. Christiansen et al., 2018, Cell 173: 792), for image restoration (M. Weigert et al., Ref. 7; W. Ouyang et al., 2018, Nat. Biotech. 36: 460).
> 2) Cell-by-cell classification such as for predicting cell fates (A. Waisman et al., 2019, Stem Cell Rep. 12: 845), for classifying cell cycle status (P. Eulenberg et al., 2017, Nat. Commun., 8: 463), for distinguishing motility behaviors of different cell types (J. Kimmel et al., 2019, IEEE/ACM Trans. Comp. Biol. Bioinf. doi: 10.1109/TCBB.2019.2919307) (probably also many other examples).

## TODO
- [x] manuscript text
